app.rb

```diff
@@ -0,0 +1,45 @@
+require 'sinatra/base'
+require_relative './lib/chitter'
+require_relative './lib/user'
+require 'pry'
+
+class ChitterManager < Sinatra::Base
+  set :session_secret, 'super secret'
+  enable :sessions
+
+  get '/' do
```

Comment:
Perhaps your URLs could be a bit more RESTful? For example, your homepage does not have a function (and is not visited in your feature tests) and the peeps page will currently have a url of something like www.chitter.com/chitter. Perhaps call it peeps and allow the user to navigate there from the homepage?